### ISSUE TITLE
The program keeps crashing on Manjaro Linux

### ISSUE THREAD

**Issue by jabbalaci** [DevToys-app/DevToys#1212]
### Current behavior

I'm testing on Manjaro Linux but it "crashes" too many times. In 5 minutes it "crashed" about 10 times. Under crash I mean that it gets stuck, at the bottom I get notified that an unhandled exception occurred and by pressing Reload I can continue working.

Here is my log file: https://pastebin.com/G0pH6GDx

I tried the v2.0.1.0 pre-release version.

### How to reproduce it (as minimally and precisely as possible)

I was just clicking around, trying out the different modules.

### Expected behavior

It shouldn't produce unhandled exceptions.

### Affected platforms

Linux

### Affected DevToys kind

DevToys (app with GUI)

### DevToys Version

v2.0.1.0 pre-release

### Relevant Assets/Logs

Here is my log file: https://pastebin.com/G0pH6GDx (a bit long)

**Comment by SnippyValson** [comment]
This happens in Ubuntu 22.04 LTS as well. I tried to debug this. But there seems to be no live logs in the terminal. I tried "devtoys -v" and "devtoys --verbose" to see if logs would be printed.

OS Information:
Operating System: Ubuntu 22.04.4 LTS              
Kernel: Linux 6.5.0-35-generic
Architecture: x86-64

**Comment by matthieu2607** [comment]
> This happens in Ubuntu 22.04 LTS as well. I tried to debug this. But there seems to be no live logs in the terminal. I tried "devtoys -v" and "devtoys --verbose" to see if logs would be printed.
> 
> OS Information: Operating System: Ubuntu 22.04.4 LTS Kernel: Linux 6.5.0-35-generic Architecture: x86-64
> 
> My log file: https://pastebin.com/mNkQKcmX

I confirm your comment

**Comment by snippyvalsonkv** [comment]
An update. I tried cloning into repo and running the Linux GUI from source. Initially I didn't run into crashes. But when I added the DevToys.Tools package using Nuget, I started running into crashes. I am not sure if adding the package was the issue. It could that I just got lucky when I ran the app before adding the DevToys.Tools package.

Also when the crashes happened there were no error logs in the terminal or the debug console. It could be that the error happens in javascript part and the error is being handled by a common exception handler, since the whole application is not crashing. It's just web UI that's crashing.

I installed the DevToys.Tools package using this command:
dotnet add package DevToys.Tools --version 2.0.0-preview.3

**Comment by SnippyValson** [comment]
An update. I tried cloning into repo and running the Linux GUI from source. Initially I didn't run into crashes. But when I added the DevToys.Tools package using Nuget, I started running into crashes. I am not sure if adding the package was the issue. It could that I just got lucky when I ran the app before adding the DevToys.Tools package.

Also when the crashes happened there were no error logs in the terminal or the debug console. It could be that the error happens in javascript part and the error is being handled by a common exception handler, since the whole application is not crashing. It's just web UI that's crashing.

I installed the DevToys.Tools package using this command:
dotnet add package DevToys.Tools --version 2.0.0-preview.3
